src/styles/createTypography.js

```diff
@@ -6,7 +6,11 @@ function round(value) {
   return Math.round(value * 1e5) / 1e5;
 }
 
-export default function createTypography(palette: Object, typography: Object | Function) {
+export default function createTypography(
+  palette: Object,
+  typography: Object | Function,
+  defaultSize: number = 16,
```

Comment:
I'm wondering if we shouldn't call it `htmlFontSize`?

Reply:
Yeah that probably makes more sense. Will fix